tensorflow/contrib/framework/BUILD

```diff
@@ -24,7 +27,42 @@ py_library(
         "python/ops/prettyprint_ops.py",
         "python/ops/variables.py",
     ],
+    data = [
+        ":python/ops/_variable_ops.so",
```

Comment:
weird, some rule is changed maybe. I run `bazel build -c opt --config=cuda //tensorflow/contrib/factorization:clustering_ops_test` and `bazel build -c opt --config=cuda //tensorflow/contrib/framework:ops_test`, they both failed.